mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -160,6 +163,14 @@ def get_feature_dependencies(model_dir):
         )
     return ""
 
+def get_vector_search_dependencies(model_dir):
+    model = _load_model(model_dir)
+    model_info = model.get_model_info()
+    index_name = model_info.flavors.get("langchain", {}).get(_DATABRICKS_DEPENDENCY_KEY, {}).get(_DATABRICKS_VECTOR_SEARCH_INDEX_NAME_KEY, None)
+    if index_name:
+        return {"index_name": index_name}
+    else:
+        return None
```

Comment:
Can this information be read directly from the MLmodel file during model version creation? Do we strictly need to extend the CreateModelVersion REST API?

Reply:
Do you know if there's existing precedent for reading from the model file in model registry? How can we access the MlModel file in model registry service?

Reply:
Yes, the model registry service has access to the MLmodel file. For example, it currently uses it to render model signature information in the UI.

@smurching can comment more for UCMR